test/unit/server/soroban/constructor_test.js

```diff
@@ -1,4 +1,4 @@
-const { Server, AxiosClient } = StellarSdk.SorobanRpc;
+const { Server, AxiosClient } = StellarSdk.Rpc;
```

Comment:
lets keep one of these as SorobanRpc so we ensure there's no backwards breakage :+1: